.gitignore

```diff
@@ -56,4 +60,12 @@ buck-out/
 *.jsbundle
 
 # CocoaPods
-/ios/Pods/
+**/ios/Pods/
+*Podfile.lock
```

Comment:
You should not be ignoring Podfile.lock. Please remove it from your gitignore.

Reply:
OK 😁